internal/logic/logic.go

```diff
@@ -121,3 +123,103 @@ func (l transactionManagementServiceLogic) NewTransaction(newTransaction model.N
 		Data:    nil,
 	}
 }
+
+func (l transactionManagementServiceLogic) DownloadTransaction(id string, cookie string) *respModel.Response {
+	transactions, _, err := l.DsSvc.Get(map[string]interface{}{"transaction_id": id}, 1, 0)
```

Comment:
`limit` & `offset` can be passed as 0 as pagination is not required here.

Reply:
done